Makefile

```diff
@@ -0,0 +1,34 @@
+
+PHONY: test verify bootstrap go-bootstrap python-bootstrap
+
+build: 
+	godep go build .
+
+# test will run the python tests using the cli 
+test: build 
+	trial -j2 dvol_python \
```

Comment:
Why do you want to do this when we are already running exactly the same tests on the line below?